dali/operators/random/coin_flip.cc

```diff
@@ -27,11 +27,11 @@ void CoinFlip::RunImpl(HostWorkspace &ws) {
 DALI_REGISTER_OPERATOR(CoinFlip, CoinFlip, CPU);
 
 DALI_SCHEMA(CoinFlip)
-  .DocStr("Produce tensor filled with 0s and 1s - results of random coin flip,"
-      " usable as an argument for select ops.")
+  .DocStr(R"code(Produces a batch of tensors filled with 0s and 1s, which is the result of random
+coin flips and is usable as an argument for select operators.)code")
```

Comment:
I'd drop the " and is usable as an argument for select operators." part

Reply:
Done